tools/dev-doctor/rootcmd.go

```diff
@@ -119,6 +123,34 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 		dockerGroupCheck{},
 	}
 
+	if *backportingChecks {
+		checks = append(checks,
+			&binaryCheck{
+				name:       "jq",
+				ifNotFound: checkError,
+			},
+			&binaryCheck{
+				name:          "python3",
+				ifNotFound:    checkError,
+				versionArgs:   []string{"--version"},
+				versionRegexp: regexp.MustCompile(`Python\s+(\d+.\d+\.\d+)`),
+				minVersion:    &semver.Version{Major: 3, Minor: 6, Patch: 0},
+			},
+			&commandCheck{
+				name:             "pygithub",
+				command:          "python3",
+				args:             []string{"-c", "from github import Github"},
+				ifFailure:        checkWarning,
+				ifSuccessMessage: "pygithub installed",
+				hint:             `Run "pip3 install --user PyGithub".`,
+			},
+			&envVarCheck{
+				name:            "GITHUB_TOKEN",
+				ifNotSetOrEmpty: checkWarning,
```

Comment:
I'd say we could even treat this as an error, since running the backporting scripts without `$GITHUB_TOKEN` set will in some cases leads to weird inconsistent states (e.g. the labels not updated on the upstream PRs) which need manual cleanup.

Reply:
Ack.